### PR TITLE
Blog Post: How we use Openzeppelin

### PR DESCRIPTION
**Pull request by nfurfaro**
# Description
The title says it all.

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Looks good, can you update the post to add it to the blog so we can publish?

**Comment by nfurfaro** [comment]
@julien51 It's in the /blog directory now. Are there any other specific updates you want to see?

**Comment by jer979** [comment]
I would love to see a referral and loyalty program baked into the product.

**Comment by julien51** [comment]
@nfurfaro yes you need to add the metadata and make sure it does actually build as part of the blog ;) Please look at other posts to see how it works!

@jer979 working on it!

File: unlock-protocol.com/blog/how-we-use-openzeppelin.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: How we use OpenZeppelin
+subTitle: Showcasing our usage of open-source tools.
+authorName: Nick Furfaro
+publishDate: February 24, 2020
+description: Openzeppelin provides the ethereum community an excellent assortment of reusable libraries and command-line tools. Here's how we use them at Unlock.
+image: /static/images/blog/Zeplin.jpeg
```
Comment (NickCuso) [inline unlock-protocol.com/blog/how-we-use-openzeppelin.md]: This is not rendering.

Comment (nfurfaro) [reply unlock-protocol.com/blog/how-we-use-openzeppelin.md]: hmm, will have a look. Not sure why not. The resource exists and the path is correct...

Comment (nfurfaro) [reply unlock-protocol.com/blog/how-we-use-openzeppelin.md]: This is the first blog post I've worked on, so I'm not sure where I should see it rendering prior to publishing?

Comment (NickCuso) [reply unlock-protocol.com/blog/how-we-use-openzeppelin.md]: ah run `yarn dev` to open the site and then click blog (top right)

Comment (nfurfaro) [reply unlock-protocol.com/blog/how-we-use-openzeppelin.md]: Ok, got it to render.

**Pull request merged**